frontend/src/components/Navbar.tsx

```diff
@@ -88,6 +90,7 @@ const Navbar: React.FC = () => {
               <Avatar alt="ProfilePic" src="">
                 {auth.user?.fullName
                   .split(' ')
+                  .filter((_, i, { length }) => [0, length - 1].includes(i)) // Take first and last name
```

Comment:
Bug: Full names that have more than 2 names include the Initials of each "name". So "Cristiano Ronaldo dos Santos Aveiro" becomes "CRDSA"
Solution: Take only the first and last words in the full name to present the Initials, assuming the middle words are middle names